[confluence] Add 8.5-8.1

ekranium opened the pull request:
Added releases 8.1 - 8.5.

ekranium committed:
Update confluence.md

Added releases 8.1 - 8.5.

BiNZGi commented on products/confluence.md:
```diff
@@ -12,11 +12,47 @@ releaseColumn: true
-    latestReleaseDate: 2022-12-13
+    eol: 2023-04-29
+    latest: "8.0.4"
+    latestReleaseDate: 2022-02-12
```
Release this year, see https://confluence.atlassian.com/doc/issues-resolved-in-8-0-4-1209869860.html:
> Released on 13 February 2023
```suggestion
    latestReleaseDate: 2023-02-12
```

BiNZGi reviewed (changes requested):
Good work, checked with https://www.atlassian.com/software/confluence/download-archives and only found a little error, see separate suggestion comment.

I found https://confluence.atlassian.com/support/atlassian-support-end-of-life-policy-201851003.html where they list the EOL date for all version. The dates seems not to correspond with the rule 6 months for non-LTS:
> 8.4 (EOL date: 6 July 2025)
> 8.3 (EOL date: 23 May 2025)
> 8.2 (EOL date: 29 Mar 2025)
> 8.1 (EOL date: 14 Feb 2025)

ekranium committed:
Update products/confluence.md

Co-authored-by: BiNZGi <BiNZGi@users.noreply.github.com>

ekranium committed:
Update confluence.md

- Updated EOL accourding to https://confluence.atlassian.com/support/atlassian-support-end-of-life-policy-201851003.html

- Changed the text explaining the difference between the LTS and non-LTS version.

BiNZGi reviewed (approved):
LGTM now -> merge
@ekranium thanks for your contribution :+1:
